server/server.go

```diff
@@ -34,15 +37,20 @@ func NewServer(config *ServerConfig, db models.DB, auth *providers.Auth, schema
 }
 
 func (s *Server) Start() {
+	config := s.config
 	h := handler.New(&handler.Config{
 		Schema:   s.schema,
 		Pretty:   true,
 		GraphiQL: true,
```

Comment:
It's actually this line that should be changed:
`GraphiQL: config.GraphiQLEnabled`

Reply:
Oh, I see. Good point 🙂